app/kumactl/cmd/inspect/inspect_dataplanes.go

```diff
@@ -139,7 +116,7 @@ func printDataplaneOverviews(now time.Time, dataplaneInsights *mesh_core.Datapla
 					meta.GetMesh(),                       // MESH
 					meta.GetName(),                       // NAME,
 					dataplane.TagSet().String(),          // TAGS
-					status,                               // STATUS
+					string(status),                       // STATUS
```

Comment:
Can we have `status.ToString()` or something similar?

Reply:
Thanks, fixed